lib/book.rb

```diff
@@ -0,0 +1,37 @@
+class Book
+  attr_reader :book,
+              :reviews
+  attr_writer :read
+
+  def initialize(book)
+    @book = book
```

Comment:
Normally, what we do is split things out. So we would have something like `@title = book[:title]`